cli/http_cache.rs

```diff
@@ -114,6 +119,30 @@ impl HttpCache {
     let serialized_headers = serde_json::to_string(&headers_map)?;
     // Cache headers
     deno_fs::write_file(&headers_filename, serialized_headers, 0o666)?;
+    self.update_manifest(url)?;
+    Ok(())
+  }
+
+  // TODO(bartlomieju): probably won't work properly when
+  // used concurrently
+  fn update_manifest(&self, url: &Url) -> Result<(), ErrBox> {
+    let manifest_filename = self
+      .location
+      .join(base_url_to_filename(url))
+      .join("manifest.json");
+    let mut manifest_map = if manifest_filename.exists() {
+      let manifest_json = fs::read_to_string(&manifest_filename)?;
+      let manifest_map: HashMap<String, String> =
+        serde_json::from_str(&manifest_json)?;
```

Comment:
Is it possible to pretty print this json?

Reply:
Do you mean write to file pretty printed JSON or actually print it (why?)?